src/SixLabors.Fonts/Tables/Cff/CffEvaluationEngine.cs

```diff
@@ -0,0 +1,748 @@
+                                    new Vector2(c1x, c1y),
+                                    new Vector2(c2x, c2y),
+                                    new Vector2(this.x, this.y));
+                            }
+
+                            this.stack.Clear();
+                            break;
+
+                        case Type2Operator1.Shortint:
+
+                            this.stack.Push(reader.ReadInt16BE());
+                            break;
+
+                        case Type2Operator1.Callgsubr:
+
+                            index = (int)this.stack.Pop() + this.globalBias;
+                            subr = this.globalSubrBuffers[index];
+
+                            if (subr.Length > 0)
+                            {
+                                this.Parse(subr);
+                            }
+
+                            break;
+
+                        case Type2Operator1.Vhcurveto:
+                        case Type2Operator1.Hvcurveto:
+
+                            phase = oneByteOperator == Type2Operator1.Hvcurveto;
+                            while (this.stack.Length >= 4)
+                            {
+                                if (phase)
+                                {
+                                    c1x = this.x + this.stack.Shift();
+                                    c1y = this.y;
+                                    c2x = c1x + this.stack.Shift();
+                                    c2y = c1y + this.stack.Shift();
+                                    this.y = c2y + this.stack.Shift();
+                                    this.x = c2x + (this.stack.Length == 1 ? this.stack.Shift() : 0);
+                                }
+                                else
+                                {
+                                    c1x = this.x;
+                                    c1y = this.y + this.stack.Shift();
+                                    c2x = c1x + this.stack.Shift();
+                                    c2y = c1y + this.stack.Shift();
+                                    this.x = c2x + this.stack.Shift();
+                                    this.y = c2y + (this.stack.Length == 1 ? this.stack.Shift() : 0);
+                                }
+
+                                this.transforming.CubicBezierTo(new Vector2(c1x, c1y), new Vector2(c2x, c2y), new Vector2(this.x, this.y));
+                                phase = !phase;
+                            }
+
+                            this.stack.Clear();
+                            break;
+
+                        case Type2Operator1.Escape:
+
+                            bool a;
+                            bool b;
+                            byte twoByteOperator = reader.ReadByte();
+                            if (twoByteOperator >= 38)
+                            {
+                                ThrowInvalidOperator(twoByteOperator);
+                            }
+                            else
```

Comment:
The intention was to remove the else nesting completely now, since `twoByteOperator < 38`, but that's just a nitpick. Leave it like it is, if you want.

Reply:
Of course, sorry! Brains elsewhere just now

Reply:
Fixed